pycbc/vetoes/sgchisq.py

```diff
@@ -137,7 +148,7 @@ def values(self, stilde, template, psd, snrv, snr_norm,
                 flow = max(kmin * template.delta_f, fcen - qwindow)
                 fhigh = fcen + qwindow
 
-                # Don't go up to Nyquist
+                # Don't filter near Nyquist
```

Comment:
for this comment I suggest: 'if any sine-gaussian tile has an upper frequency limit above 0.9*Nyquist, abort the chisq calculation and return 1 instead'
so, darn, the inline review doesn't allow me to get at lines that weren't changed.  anyway, on line 158 you have the typo 'sing-gaussian'.